addons/website/views/snippets/snippets.xml

```diff
@@ -950,7 +950,7 @@
     </div>
 
     <div data-js="TopMenuVisibility"
-         data-selector="[data-main-object^='website.page('] #wrapwrap > header"
+         data-selector="[data-main-object]:has(input.o_page_option_data[name='header_overlay']):has(input.o_page_option_data[name='header_visible']) #wrapwrap > header"
```

Comment:
Just to be sure, are `header_overlay` and `header_visible` always present at the same time ? Same question for `header_color` and `header_text_color` 
Because I think that if one of them is not here, the selector won't work :thinking:

Reply:
AFAIK yes - because they used to only be inside `website.page`, and now they always all are in the mixin.
I based the selector on the values needed by the options that appear in case of match.
So that, if some type does not have both needed, it does not qualify for those options (which would crash upon use because the field would be missing)